### PR TITLE
Feature/authentication cloud express

### PR DESCRIPTION
**Pull request by danilobispo**
Authentication using pki express and TSP

### CODE REVIEW

File: java/springmvc/src/main/java/com/lacunasoftware/pkisuite/controller/AuthenticationCloudExpressController.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	 * parameter of the method discoverByCpfAndStartAuth(). This parameter will be recovered in the
+	 * next action.
+	 */
+	@PostMapping("/authentication-cloud-express/discover")
+	public ModelAndView discover(
+		@RequestParam(value="cpf") String cpf
+	) throws IOException {
+        // Get an instance of the Authentication class, responsible for contacting PKI Express to
+		// start and complete the authentication operation.
+        Authentication auth = new Authentication();
```
Comment (medisco) [inline java/springmvc/src/main/java/com/lacunasoftware/pkisuite/controller/AuthenticationCloudExpressController.java:76]: Can you fix the indentation of the flies? It seems to have an mixture of tab and spaces.

File: java/springmvc/src/main/java/com/lacunasoftware/pkisuite/controller/AuthenticationCloudExpressController.java [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+		// At this point, you have assurance that the certificate is valid. Now, you'd typically
+		// query your database for a user that matches one of the certificate's fields, such as
+		// userCert.getEmailAddress() or userCert.getPkiBrazil().getCpf() (the actual field to be
+		// used as key depends on your application's business logic) and set the user as
+		// authenticated with whatever web security framework your application uses. For
+		// demonstration purposes, we'll just render the user's certificate information.
+		model.addAttribute("userCert", result.getCertificate());
+
+		// Render the authentication succeeded page (templates/authentication-success.html)
+		return "authentication-express/success";
```
Comment (medisco) [inline java/springmvc/src/main/java/com/lacunasoftware/pkisuite/controller/AuthenticationCloudExpressController.java:179]: Here, it's returning a view from other example. Shouldn't return something like: `authentication-cloud-express/success`?
I also noticed this view doesn't exist. There is only the `failed` page.

**Review by medisco — CHANGES_REQUESTED** [review]
Add some comments

File: java/springmvc/src/main/java/com/lacunasoftware/pkisuite/controller/AuthenticationCloudExpressController.java (hunk 1 above)
Comment (danilobispo) [reply java/springmvc/src/main/java/com/lacunasoftware/pkisuite/controller/AuthenticationCloudExpressController.java]: Made the changes in indentation in the next commits

File: java/springmvc/src/main/java/com/lacunasoftware/pkisuite/controller/AuthenticationCloudExpressController.java (hunk 2 above)
Comment (danilobispo) [reply java/springmvc/src/main/java/com/lacunasoftware/pkisuite/controller/AuthenticationCloudExpressController.java]: Please check https://github.com/LacunaSoftware/PkiSuiteSamples/pull/104/commits/ba06c6680d8c365ab41ef3452979b2df861e5666

**Review by medisco — APPROVED** [review]
LGTM!

**Pull request merged**